Tests/ViewControllers/BrickViewControllerTests.swift

```diff
@@ -51,6 +51,22 @@ class BrickViewControllerTests: XCTestCase {
         }
     }
 
+    private var testBrickViewControllerNibName: String {
+        #if os(tvOS)
+        return "TestBrickViewController-tvOS"
+        #else
+        return "TestBrickViewController"
+        #endif
+    }
+
+    private var testBrickViewControllerStoryboard: UIStoryboard {
```

Comment:
Could this getter just be 
```
private var testBrickViewControllerStoryboard: UIStoryboard {
    return UIStoryboard(name: testBrickViewControllerNibName, bundle: Bundle(for: type(of: self)))
}
```

Reply:
Maybe also rename `testBrickViewControllerNibName` to something that works for both cases.

Reply:
Thanks for the look. I think we’re gonna roll with this for now, maybe when we next clean up this whole file we can make it better…